syft/frameworks/torch/tensors/interpreters/large_precision.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import torch
+
+from syft.frameworks.torch.overload_torch import overloaded
+from syft.frameworks.torch.tensors.interpreters import AbstractTensor
+
+
+class LargePrecisionTensor(AbstractTensor):
+    """LargePrecisionTensor allows handling of numbers bigger than LongTensor
+
+    Some systems using Syft require larger types than those supported natively. This tensor type supports arbitrarily
+    large values by packing them in smaller ones.
+    These smaller values are of type `internal_type`. The large value is defined by `virtual_prec`
+    """
+
+    def __init__(
+        self,
+        owner=None,
+        id=None,
+        tags=None,
+        description=None,
+        internal_type=torch.int32,
+        virtual_prec=0,
+    ):
+        """Initializes a LargePrecisionTensor.
+
+        Args:
+            owner (BaseWorker): An optional BaseWorker object to specify the worker on which
+                the tensor is located.
+            id (str or int): An optional string or integer id of the LargePrecisionTensor.
+            tags (list): list of tags for searching.
+            description (str): a description of this tensor.
+            internal_type (int): The precision this tensor will be transformed to internally. In bits.
+            virtual_prec (int): The virtual precision required by the caller. In bits.
```

Comment:
Maybe even further clairfy `internal_type` : you could say that the storage of the large tensor is using small tensors of size `internal_type` bits, or smthg equivalent

Reply:
Also, internal_type in now a type, you should modify the type info